lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -395,11 +395,15 @@ def insert_at_position(position, raise_exception_if_save_fails=false)
             if in_list?
               old_position = current_position
               return if position == old_position
-              # temporary move after bottom with gap, avoiding duplicate values
-              # gap is required to leave room for position increments
-              # positive number will be valid with unique not null check (>= 0) db constraint
-              temporary_position = bottom_position_in_list + 2
-              set_list_position(temporary_position, raise_exception_if_save_fails)
+
+              if sequential_updates?
```

Comment:
Not sure if I should check `sequential_updates`, or add a new option to bypass this logic. Let me know what you think @brendon.